samples/node/routes/weather.js

```diff
@@ -11,19 +11,37 @@ const sha1 = require('sha1');
 exports.requestCards = function(req, res) {
     console.log('cards called: ', req.body);
 
-    if (!req.body.tokens) {
-        res.status(400).send("Missing tokens field");
+    let tokenZip = req.body.tokens && req.body.tokens.zip;
+    let configZip = req.body.config && req.body.config.defaultZip;
+
+    console.log(`tokenZip: ${tokenZip}`);
```

Comment:
Do you need to log these if the whole request body was already logged (line 12)?

Reply:
I don’t have to, I thought about cutting some of the logging out.

I was logging just to clearly show what values were parsed out for each.

Reply:
```
cards called:  { config: { defaultZip: '30332' },
  zips: [ '30080', '30076' ],
  connector_id: '46a42fb8-1097-427a-a8b1-491c8b53aba7',
  object_type: 'card' }
tokenZip: 30080,30076
configZip: 30332
Valid Input: 30080,30076
Zips: 30080,30076
```

and

```
cards called:  { config: { defaultZip: '30332' },
  connector_id: '46a42fb8-1097-427a-a8b1-491c8b53aba7',
  object_type: 'card' }
tokenZip: undefined
configZip: 30332
Valid Input: 30332
Zips: 30332
```

seem a little excessive, but it's not worth waiting another day or 2.